package.json

```diff
@@ -13,7 +13,7 @@
     "lint:fix": "next lint --fix",
     "prettier": "prettier -w \"{src,cypress,mocks,scripts}/**/*.{ts,tsx,css,js}\"",
     "fix": "yarn lint:fix && ts-prune && yarn prettier",
-    "test": "cross-env TZ=CET DEBUG_PRINT_LIMIT=30000 jest --watchAll",
```

Comment:
Why remove watchAll?

Reply:
Because in the `dev` & `main` branches the `test` command do not have the ` --watchAll` flag. I added it by mistake because for me its more confortable work with that flag enabled.

see: https://github.com/safe-global/web-core/blob/dev/package.json#L16